x-pack/plugins/serverless_search/kibana.jsonc

```diff
@@ -7,11 +7,7 @@
     "id": "serverlessSearch",
     "server": true,
     "browser": true,
-    "configPath": [
```

Comment:
seems like a formatting conflict with these changes. are you missing prettier?

Reply:
@TattdCodeMonkey I'm using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) in VS Code, which formats the JSONC files in this way. Which formatter do we use for Kibana development?

Reply:
I'm not sure, but ideally it shouldn't make these changes given that there was not actual change to the values.